rpc/types_block.go

```diff
@@ -123,6 +123,28 @@ type PendingBlock struct {
 	BlockTransactions
 }
 
+// encoding/json doesn't support inlining fields
+type BlockWithReceipts struct {
+	BlockStatus string `json:"status"`
+	BlockHeader
+	BlockBodyWithReceipts
+}
+
+type BlockBodyWithReceipts struct {
+	Transactions []TransactionWithReceipt `json:"transactions"`
+}
+
+type TransactionWithReceipt struct {
+	Transaction Transaction        `json:"transaction,omitempty"`
+	Receipt     TransactionReceipt `json:"receipt"`
```

Comment:
This field's schema is not yet finalized, let's wait that to be finalized.

Reply:
Receipts were updated in rc1. I will incorporate the rc-1 changes in this PR

Reply:
Receipts seem to be finalised in rc-1.